### PR TITLE
[WIP] Add new recipe for Python package SPICE-HSI

### PR DESCRIPTION
**Pull request by nfahlgren**
SPICE-HSI (Sparsity Promoting Iterated Constrained Endmember Detection in Hyperspectral Imagery) is a package for extracting endmembers (pure signature of a material) from hyperspectral image data.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/spice-hsi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5660ce422871671896c8bb0084acbb22d7ff2f785e0aa49292ccc8902fb8aebd
+
+build:
+  number: 0
+  skip: true  # [py>3.7]
+
```
Comment (npavlovikj) [inline recipes/spice-hsi/meta.yaml:15]: I think this recipe can be noarch, https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds, so please add `noarch: python` to the build section. 
To skip python 3.8, in the `requirements` you can then add `python  <3.8`

File: recipes/spice-hsi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+requirements:
+  host:
+    - pip
```
Comment (npavlovikj) [inline recipes/spice-hsi/meta.yaml:18]: python should be `host` dependency as well.

**Comment by nfahlgren** [comment]
Thanks @npavlovikj, I appreciate it!

**Comment by nfahlgren** [comment]
@npavlovikj, we updated SPICE to remove some unused stuff in the `setup.py` file that was causing issues with the package build. Mind taking a look again? Thanks!

**Comment by npavlovikj** [comment]
Great @nfahlgren , the recipe looks good now! Thank you for contributing 🎉

**Pull request merged**